packages/mobile/e2e/global-setup.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs-extra')
+const { execSync } = require('child_process')
+const detox = require('detox')
+
+async function globalSetup() {
+  const config = resolveSelectedConfiguration() || {}
+  downloadTestButlerAPKIfNeeded(config)
+  await detox.globalInit()
+}
+
+function downloadTestButlerAPKIfNeeded(config) {
+  if (isAndroidConfig(config)) {
+    downloadTestButlerAPK()
+  }
+}
+
+function downloadTestButlerAPK() {
+  const version = '2.2.1'
+  const artifactUrl = `https://repo1.maven.org/maven2/com/linkedin/testbutler/test-butler-app/${version}/test-butler-app-${version}.apk`
+  const filePath = `./e2e/test-butler-app.apk`
```

Comment:
these feel like they should be in a config file somewhere but i remember you saying that we will make a global change for it at some point. maybe these should live at the top of the file, as constants?

Reply:
[Test Butler](https://github.com/linkedin/test-butler/releases) doesn't release often. This code was largely from [this example](https://github.com/wix/Detox/tree/master/examples/demo-react-native-jest/e2e).